docs/exclusive-content.md

```diff
@@ -212,3 +204,41 @@ If you see the source files instead of the example, click **View App** in the bo
+The embedded JavaScript script will parse all the exclusive content <code>div</code> tags and include the proxy payment pointer in the web page's header (if there are multiple, it will select one at random). If Web Monetization is enabled by the User, receipts can now be obtained from the <code>monetizationprogress</code> events. The receipts, together with the payment pointer and the encrypted verifier endpoint, are submitted to the Exclusive Content Generator, which derives the encryption key and decrypts the verifier endpoint. If the Exclusive Content Generator is able to verify the receipts with the STREAM receipt verifier, it shares the encryption key with the User's client, who is now able to decrypt the content and display it.
+
+[![](https://mermaid.ink/img/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG5hdXRvbnVtYmVyXG5Vc2VyJ3MgQ2xpZW50IC0-PiBDcmVhdG9yJ3MgV2FsbGV0OiBzdHJlYW1zIHBheW1lbnRzXG5DcmVhdG9yJ3MgV2FsbGV0IC0-PiBVc2VyJ3MgQ2xpZW50OiBTVFJFQU0gcmVjZWlwdHNcblVzZXIncyBDbGllbnQgLT4-IEV4Y2x1c2l2ZSBDb250ZW50IEdlbmVyYXRvcjogQ3JlYXRvcidzIHBheW1lbnQgcG9pbnRlciwgZW5jcnlwdGVkIHZlcmlmaWVyIGVuZHBvaW50LCBTVFJFQU0gcmVjZWlwdFxuRXhjbHVzaXZlIENvbnRlbnQgR2VuZXJhdG9yIC0-PiBFeGNsdXNpdmUgQ29udGVudCBHZW5lcmF0b3I6IGRlcml2ZXMga2V5LCBkZWNyeXB0cyB2ZXJpZmllciBlbmRwb2ludCwgdmVyaWZpZXMgcmVjZWlwdFxuRXhjbHVzaXZlIENvbnRlbnQgR2VuZXJhdG9yIC0-PiBVc2VyJ3MgQ2xpZW50OiBrZXlcblVzZXIncyBDbGllbnQgLT4-IFVzZXIncyBDbGllbnQ6IGRlY3J5cHRzIGNvbnRlbnQiLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG5hdXRvbnVtYmVyXG5Vc2VyJ3MgQ2xpZW50IC0-PiBDcmVhdG9yJ3MgV2FsbGV0OiBzdHJlYW1zIHBheW1lbnRzXG5DcmVhdG9yJ3MgV2FsbGV0IC0-PiBVc2VyJ3MgQ2xpZW50OiBTVFJFQU0gcmVjZWlwdHNcblVzZXIncyBDbGllbnQgLT4-IEV4Y2x1c2l2ZSBDb250ZW50IEdlbmVyYXRvcjogQ3JlYXRvcidzIHBheW1lbnQgcG9pbnRlciwgZW5jcnlwdGVkIHZlcmlmaWVyIGVuZHBvaW50LCBTVFJFQU0gcmVjZWlwdFxuRXhjbHVzaXZlIENvbnRlbnQgR2VuZXJhdG9yIC0-PiBFeGNsdXNpdmUgQ29udGVudCBHZW5lcmF0b3I6IGRlcml2ZXMga2V5LCBkZWNyeXB0cyB2ZXJpZmllciBlbmRwb2ludCwgdmVyaWZpZXMgcmVjZWlwdFxuRXhjbHVzaXZlIENvbnRlbnQgR2VuZXJhdG9yIC0-PiBVc2VyJ3MgQ2xpZW50OiBrZXlcblVzZXIncyBDbGllbnQgLT4-IFVzZXIncyBDbGllbnQ6IGRlY3J5cHRzIGNvbnRlbnQiLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)
+
+### Interactive example
+
+This example page will request the decryption key for each piece of exclusive content if Web Monetization is enabled. If it is not enabled, it will say "🔒 This content is exclusive for users with Web Monetization enabled."
+
+The example does require you to have Web Monetization enabled in your browser, hence you need to [open the page](https://exclusive-content-demo.glitch.me/) with Web Monetization enabled.
+
+If you see the source files instead of the example, click **View App** in the bottom right.
+
+<div class="glitch-embed-wrap" style={{ height: '420px', width: '100%' }}>
+  <iframe
+    src="https://glitch.com/embed/#!/embed/exclusive-content-demo?path=index.html&previewSize=100"
```

Comment:
fwiw
typo in the glitch app: `Mutlimedia Link`
and the three parts unlock, but I get this [error from the coil extension](https://github.com/coilhq/web-monetization-projects/blob/a367f8dd1c387c6d3739ab5ec0fda38a71bfaf88/packages/coil-extension/src/content/services/ContentScript.ts#L106-L109):
```
content.js:21 <iframe> (or one of its ancestors) is not authorized to allow web monetization, https://cinnamon.video/watch?v=439021293652673898
```

Reply:
I added `allow="monetization"` to the iframe but it is still not  a great example since the cinnamon payment pointer doesn't seem to work outside of cinnamon or with the verifier.

Reply:
I changed the example to be a video from archive.org using their uphold payment pointer.